pkg/api/pluginproxy/ds_proxy_test.go

```diff
@@ -527,6 +527,72 @@ func TestDSRouteRule(t *testing.T) {
 					"important_cookie=important_value")
 			})
 		})
+
+		Convey("When datasource hasn't been updated, encrypted JSON should be fetched from cache", func() {
```

Comment:
Not sure it makes sense to test the caching of decrypted data here. Should have tests in datasource_cache_test.go